gui/pages/kernel_options.go

```diff
@@ -7,12 +7,13 @@ package pages
 import (
 	"github.com/clearlinux/clr-installer/gui/common"
 
+	"strings"
```

Comment:
Please make "strings" first.
The imports should always be

1.  Built-in Language
2. Vendor Code
3. This application's packages

Reply:
Done